lib/flame_splash_screen.dart

```diff
@@ -0,0 +1,75 @@
+library flame_splash_screen;
+
+import 'package:flutter/material.dart';
+
+class FlameSplashScreenWidget extends StatefulWidget {
+  final bool darkTheme;
+  final Duration fadeInDuration;
+  final Duration waitDuration;
+  final Duration fadeOutDuration;
+
+  final String navigateTo;
+
+  FlameSplashScreenWidget({
+    @required this.navigateTo,
+    this.darkTheme = false,
+    this.fadeInDuration = const Duration(seconds: 1),
+    this.waitDuration = const Duration(seconds: 1),
+    this.fadeOutDuration = const Duration(seconds: 1),
+  });
+
+  @override
+  State<StatefulWidget> createState() => _FlameSplashScreenWidgetState();
+}
+
+class _FlameSplashScreenWidgetState extends State<FlameSplashScreenWidget> {
+
+  bool _visible = false;
+
+  @override
+  void initState() {
+    super.initState();
+
+    Future.delayed(widget.waitDuration).then((_) {
+      setState(() {
+        _visible = true;
+      });
+
+      Future.delayed(widget.fadeInDuration).then((_) {
+        setState(() {
+          _visible = false;
+        });
+      });
+    });
+
+    Future.delayed(
+        widget.waitDuration +
+        widget.fadeInDuration +
+        widget.fadeOutDuration
+    ).then((_) {
+      Navigator.of(context).pushReplacementNamed(widget.navigateTo);
+    });
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return Container(
+        color: widget.darkTheme ? Color(0xFF000000) : Color(0xFFFFFFFF),
+        child: Column(
+            mainAxisAlignment: MainAxisAlignment.center,
+            children: [
+              AnimatedOpacity(
+                  opacity: _visible ? 1 : 0,
+                  duration: widget.fadeInDuration,
+                  child: Image(
+                      width: 300,
+                      image: AssetImage(
+                          'assets/flame-logo-${widget.darkTheme ? 'black' : 'white'}.gif',
```

Comment:
can't we have a single transparent gif? and the the color can be configured as rgb, with preset options

Reply:
That would be a huge amount of work, since this came from the css animation that we have on the site, I would have to edit the gif frame by frame, and that would take too much time.

My original idea was to have a Flare animation for this, but since we don't have the flare animation yet, I tried the next best, and possible thing.